modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/context/index.jsx

```diff
@@ -68,25 +83,29 @@ const AppContextProvider = ({children}) => {
 					({name}) => name === 'Administrator'
 				);
 
-				const isPartner = data.userAccount?.organizationBriefs?.filter(
-					({name}) => (
-						name !== 'Account Access EU' &&
-						name !== 'Account Access US' && 
-						name !== 'Liferay Staff' && 
-						name !== 'Systems - Provisioning')
-				).length > 0;
+				const isPartner =
+					data.userAccount?.organizationBriefs?.filter(
+						({name}) =>
+							name !== 'Account Access EU' &&
+							name !== 'Account Access US' &&
+							name !== 'Liferay Staff' &&
+							name !== 'Systems - Provisioning'
+					).length > 0;
 
 				const isStaff = data.userAccount?.organizationBriefs?.some(
 					(organization) => organization.name === 'Liferay Staff'
 				);
 
 				const userAccount = {
 					...data.userAccount,
+
```

Comment:
why is there extra new line spaces around this param?

Reply:
I'll adjust, thanks